STUDIO/org.openl.rules.webstudio/webapp/WEB-INF/spring/webstudio-true.xml

```diff
@@ -17,6 +17,8 @@
 
     <bean id="currentUserInfo" class="org.openl.rules.webstudio.security.CurrentUserInfo"/>
 
+    <bean id="mailSender" class="org.openl.rules.webstudio.mail.MailSender"/>

```

Comment:
Please, use annotation. I want to get rid of XML configuration as much as it is possible.